src/Files.App/Dialogs/AddBranchDialog.xaml.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Files.App.Dialogs
 {
-	public sealed partial class AddBranchDialog : ContentDialog, IDialog<AddBranchDialogViewModel>
+	public sealed partial class AddBranchDialog : ContentDialog, IDialog<AddBranchDialogViewModel>, IRealTimeControl
```

Comment:
FYI, While this looks somewhat weird just for this but I'm planning to create an all-in-one service for ContentDialog & DynamicDialog to generate every dialog we use. The dialog contents will be Page class derivatives instead of the ContentDialog class.

For example:
```c#
IAppDialogService service = new();
var res = service.Show_AddBranchDialog();
```